portfolio/src/main/java/com/google/sps/servlets/CommentServlet.java

```diff
@@ -52,11 +56,20 @@ public void doPost(HttpServletRequest request, HttpServletResponse response) thr
     // Get comments from the form
     String name = getParameter(request, /*name=*/"name", /*defaultValue=*/"");
     String comment = getParameter(request, /*name=*/"comment", /*defaultValue=*/"No comments");
+    long timestamp = System.currentTimeMillis();
     // TODO: validate request parameters
 
     // Modify the state of server.
     addComment(name, comment);
 
+    Entity commentEntity = new Entity("Comment");
+    commentEntity.setProperty("name", name);
+    commentEntity.setProperty("comment", comment);
+    commentEntity.setProperty("time", timestamp);
+
+    DatastoreService datastoreService = DatastoreServiceFactory.getDatastoreService();
+    datastoreService.put(commentEntity);
```

Comment:
Now that we put comment into the data store, do we load them out when servlet starts?

Reply:
It is in the another PR haha